### ISSUE TITLE
Does not work with "Reduce transparency" option

### ISSUE THREAD

**Issue by jonasengelmann** [igorkulman/ChangeMenuBarColor#59]
When choosing the "Reduce transparency" option under System Preferences > Accessibility > Display, it does not work, it just stays gray.  I am on mac os 12.0 and I used the precompiled version 1.0.0 which otherwise works fine.

**Comment by igorkulman** [comment]
It is a "feature" of macos, the menu bar is not transparent but solid gray so no way to make it work.
